frontend/packages/ceph-storage-plugin/src/components/modals/add-capacity-modal/add-capacity-modal.tsx

```diff
@@ -207,37 +207,42 @@ export const AddCapacityModal = (props: AddCapacityModalProps) => {
           />
         ) : (
           <>
-            <FormGroup
-              className="pf-u-py-sm"
-              fieldId="request-size"
-              id="requestSize__FormGroup"
-              label={t('ceph-storage-plugin~Raw Capacity')}
-              labelIcon={<FieldLevelHelp>{requestedCapacityTooltip(t)}</FieldLevelHelp>}
-            >
-              <TextInput
-                isDisabled
-                id="request-size"
-                className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
-                type="number"
-                name="requestSize"
-                value={osdSizeWithoutUnit}
-                aria-label="requestSize"
-                data-test-id="requestSize"
-              />
-              {provisionedCapacity && (
+            {provisionedCapacity ? (
+              <FormGroup
+                className="pf-u-py-sm"
+                fieldId="request-size"
+                id="requestSize__FormGroup"
+                label={t('ceph-storage-plugin~Raw Capacity')}
+                labelIcon={<FieldLevelHelp>{requestedCapacityTooltip(t)}</FieldLevelHelp>}
+              >
+                <TextInput
+                  isDisabled
+                  id="request-size"
+                  className={classNames('pf-c-form-control', 'ceph-add-capacity__input')}
+                  type="number"
+                  name="requestSize"
+                  value={osdSizeWithoutUnit}
+                  aria-label="requestSize"
+                  data-test-id="requestSize"
+                />
                 <TextContent className="ceph-add-capacity__provisioned-capacity">
                   {' '}
                   {t('ceph-storage-plugin~x {{ replica, number }} replicas =', {
                     replica,
                   })}{' '}
                   <strong data-test="provisioned-capacity">{provisionedCapacity}&nbsp;TiB</strong>
                 </TextContent>
-              )}
+                <TextContent className="pf-u-font-weight-bold pf-u-secondary-color-100 ceph-add-capacity__current-capacity">
+                  {t('ceph-storage-plugin~Currently Used:')}&nbsp;
+                  {currentCapacity}
+                </TextContent>
+              </FormGroup>
+            ) : (
```

Comment:
Let's move it to the `helperText` field of FormGroup OR as a separate non-form component.
The first options seem good.